bin/ansible-lint

```diff
@@ -78,7 +78,7 @@ def main(args):
     parser.add_option('--force-color', dest='colored',
                       default=True,
                       action='store_true',
-                      help="Try force colored output")
+                      help="Try force colored output [default]")
```

Comment:
force color should not the default. The default is whatever the running process supports - force color is only needed if the running process (e.g. Jenkins) is not a TTY but does support colour.